src/components/Header/guest-user-content/index.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { Link } from "react-router-dom";
+
+import { IconButton } from "@mui/material";
+import { t } from "i18next";
+
+import { ShopIcon } from "~/assets/icons/shop-cart-icon.tsx";
+import { AppRoute, headerLoginLinks } from "~/libs/constants/index.ts";
+
+import { NavHeader } from "../nav-header/index.tsx";
+import { ButtonsContainer, IconsSection, StyledBadge } from "../styles.ts";
+import { StyledButtonLogIn, StyledButtonSignUp } from "./styles.ts";
+
+const GuestUserContent: React.FC = () => {
+	return (
+		<>
+			<NavHeader links={headerLoginLinks} />
+			<IconsSection>
+				<ButtonsContainer>
+					<StyledButtonLogIn href={AppRoute.SIGN_IN}>
+						{t("header.buttons.LogIn")}
+					</StyledButtonLogIn>
+					<StyledButtonSignUp href={AppRoute.SIGN_UP}>
+						{t("header.buttons.SignUp")}
+					</StyledButtonSignUp>
+				</ButtonsContainer>
+				<StyledBadge
+					// Pass here you product request count or order count
+					badgeContent={2}
+				>
```

Comment:
Is this a comment for further adjustments or for what?

Reply:
in this case better to add //TODO Pass here you product request count or order count
its will be to easy to check all comments that need be replaced, by word TODO